tests/CMakeLists.txt

```diff
@@ -44,3 +44,8 @@ add_executable(Sketcher_tests_run)
 add_subdirectory(src/Mod/Sketcher)
 target_include_directories(Sketcher_tests_run PUBLIC ${EIGEN3_INCLUDE_DIR})
 target_link_libraries(Sketcher_tests_run gtest_main ${Google_Tests_LIBS} Sketcher)
+
+include(GoogleTest)
+# discovers tests by asking the compiled test executable to enumerate its tests
+gtest_discover_tests(Tests_run) 
+gtest_discover_tests(Sketcher_tests_run) 
```

Comment:
can you explain why these are added? Does not seem immediately related to vscode integration

Reply:
and thats where ctest and gtest "communicate" (see my other comment)

Reply:
Does it hcange anything for those not using vscode?